me.jtheoof.swappy.yaml

```diff
@@ -0,0 +1,32 @@
+id: me.jtheoof.swappy
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: swappy
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=home
```

Comment:
Seems like the program prefers `$XDG_CONFIG_HOME/Desktop` first over `$HOME/Desktop` as the save location, `$XDG_CONFIG_HOME` is set inside the sandbox. Where does the screenshot end up without save_dir in config file?


Also, maybe restrict this to `$HOME/Desktop` or `$HOME/Pictures` by default. That's the usual and default save locations. If someone wants a custom location, they can add the permission.

Reply:
This is the code:
https://github.com/jtheoof/swappy/blob/596b9a8b17234303a9d4d2ea1d3107cb5442253e/src/config.c#L29-L32

Since `$XDG_DESKTOP_DIR` is not in the sandbox and `$XDG_CONFIG_HOME/Desktop` is I think a **bug** (it usually doesn't exist), it will save them in `$HOME/Desktop`. (tested behavior)

---

I didn't restrict it to `$HOME/Pictures` because it doesn't just save images, it also open images. It can be used by directly passing an image to stdin, or by specifying an image path (there is a .desktop file to open `png` and `jpen`).

Reply:
Can I specify the `--filesystem=home` as `:ro` and `--filesystem=$HOME/Pictures` as `:rw`?

Reply:
> it will save them in $HOME/Desktop. (tested behavior)

Yea, looks like, it saves to there. I didn't check there. xdg-desktop is a weird location to save the images, it should use xdg-pictures instead.

> Can I specify the --filesystem=home as :ro and --filesystem=$HOME/Pictures as :rw?

Yea, that seems fine: `--filesystem=xdg-desktop` and `--filesystem=home:ro`. $HOME/Pictures is entirely unused.